lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -813,3 +812,58 @@ def get_next_line(fh):
         # Discard the rest of the line
         fh.readline()
     return line.strip()
+
+class VtkXml(GenericXml):
+    """Format for defining VTK (XML based) and its sub-datatypes. https://docs.vtk.org/en/latest/design_documents/VTKFileFormats.html"""
+
+    edam_data = "edam:data_3671"
+    edam_format = "edam:format_3621"
+    file_ext = "vtkxml"
+
+    MetadataElement(name="vtk_version", default=None, desc="Vtk version", readonly=True, optional=True, visible=True)
```

Comment:
Maybe add a comment that these 3 elements are also available for the legacy vtk datatypes.